ansible_collections/arista/avd/molecule/eos_cli_config_gen/inventory/host_vars/router-ospf.yml

```
@@ -57,6 +57,9 @@ router_ospf:
           initial: 100
           min: 200
           max: 300
+      eos_cli:
+        area 5 not-so-stubby lsa type-7 convert type-5
+        network 3.2.4.1 0.255.255.255 area 3.2.4.1
```

This is what will appear in the config on the device (EOS 4.27)
```suggestion
        network 3.0.0.0/8 area 3.2.4.1
```

Though, yes the configuration supplied will be accepted by the device.